source/php/ComponentsJs.php

```diff
@@ -51,7 +51,10 @@ public function createComponents() //:void
                     'key' => $this->getKey($view),
                     'html' => $this->renderView(
                         $view,
-                        ['lang' => (object) $this->lang]
+                        [
+                            'lang' => (object) $this->lang,
+                            'icon' => get_field('like_icon', 'option') ?? 'favorite'
```

Comment:
This is not scalable. Solution must tie specific data to each view. May be done by checking the view-name, or with a filter.